plan_test.go

```diff
@@ -118,18 +118,18 @@ func TestQueryPlanABA2(t *testing.T) {
 				"SelectionSet": "{ _id: id compTitles(limit: 42) { id compTitles(limit: 666) { id } } }",
 				"InsertionPoint": ["movies"],
 				"Then": [
-				  {
+					{
 					"ServiceURL": "A",
 					"ParentType": "Movie",
 					"SelectionSet": "{ _id: id title }",
-					"InsertionPoint": ["movies", "compTitles"],
+					"InsertionPoint": ["movies", "compTitles", "compTitles"],
```

Comment:
This change to an existing test is functionally identical to its old assertion. The only thing that's changed here is the array order of the resulting steps, which is benign to the execution process.